avaframe/com1DFA/com1DFA.py

```diff
@@ -1988,7 +1997,40 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, simHash=""):
     resultsDFPath = pathlib.Path(cfgGen["avalancheDir"], "Outputs", "com1DFA", "resultsDF_%s.csv" % simHash)
     resultsDF.to_csv(resultsDFPath)
 
-    return Tsave, particlesList, fieldsList, infoDict
+    if cfg["EXPORTS"].getboolean("exportData"):
+        # exportFields(cfg, Tsave, fieldsList, dem, outDir, cuSimName)
+        exportFields1(cfg, t, fields, dem, outDir, cuSimName, TSave="final")
+        
+        # export particles dictionaries of saving time steps
+        savePartToPickle(particles, outDirData, cuSimName)
+    else:
+        # fetch contourline info
+        contDictXY = outCom1DFA.fetchContCoors(
+            dem["header"], fields[cfg["VISUALISATION"]["contourResType"]], cfg["VISUALISATION"], cuSimName
+        )
+
+    # save contour line for each sim
+    contDictXY = outCom1DFA.fetchContCoors(
+        dem["header"], fields[cfg["VISUALISATION"]["contourResType"]], cfg["VISUALISATION"], cuSimName
+    )
+    outDirDataCont = outDir / "contours"
+    fU.makeADir(outDirDataCont)
+    saveContToPickle(contDictXY, outDirDataCont, cuSimName)
```

Comment:
Why do we need the contourlines for each sim?

Reply:
this functionality we have if we set the exportData = False as default, so that if no peakFiles are saved at least the contourlines are produced to have an idea of the results, this is for example helpful if a large number of sims is produced but not all the data shall be saved